third_party/go/BUILD

```diff
@@ -20,6 +20,65 @@ package(default_visibility = ["PUBLIC"])
 #     revision = "v6.8.2",
 # )
 
+go_get(
+    name = "go-jira",
+    get = "github.com/andygrunwald/go-jira",
+    licences = ["MIT"],
+    revision = "v1.12.0",
+    deps = [
+        ":pkg_errors",
+        ":go-querystring",
+        ":jwt-go",
+        ":structs",
+        ":tgo",
+    ],
+)
+
+go_get(
+    name = "go-querystring",
+    get = "github.com/google/go-querystring/query",
+    licences = [
+        "BSD-3-Clause",
+    ],
+    revision = "v1.0.0",
+)
+
+go_get(
+    name = "jwt-go",
+    get = "github.com/dgrijalva/jwt-go",
+    licences = ["mit"],
+    revision = "06ea1031745cb8b3dab3f6a236daf2b0aa468b7e",
+)
+
+go_get(
+    name = "jwt",
```

Comment:
do we need this? can't see it used anywhere

Reply:
That's true; I had a feeling `jwt-go` (which is needed) depends on `jwt`, but it's not the case. Thank you